parcels/grid.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, lon, lat, time, time_origin, mesh):
         self.meridional_halo = 0
         self.lat_flipped = False
         self.defer_load = False
+        self.lonlat_minmax = np.array([np.min(lon), np.max(lon), np.min(lat), np.max(lat)])
```

Comment:
use `np.nanmin` and `np.nanmax`? 
(this is related to another issue we have with grids where some lon/lat over land are set to Nan)